### ISSUE TITLE
add code for downloading resources other than videos

### ISSUE THREAD

**Issue by moghya** [moghya/nptelDl#1]

**Comment by DevCabin** [comment]
I checked out the site, the other format I see is embedded PDF documents - is that what you would like to be able to download besides video?

**Comment by moghya** [comment]
Exactly, some courses have pdfs some don't have to, it would be better of
we let user know about everything available for download and then user can
download as per his needs

On 24-Oct-2017 1:53 AM, "George Featherstone" <notifications@github.com>
wrote:

I checked out the site, the other format I see is embedded PDF documents -
is that what you would like to be able to download besides video?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/moghya/nptelDl/issues/1#issuecomment-338785059>, or mute
the thread

**Comment by moghya** [comment]
@DevCabin have you done any kind of work on this ?
